resources/stubs/Relation/HasMany.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * Spiral Framework.
+ *
+ * @license   MIT
+ * @author    Anton Titov (Wolfy-J)
+ */
```

Comment:
Same. Invalid docblock. 

P.S. There are several more files that contain the same docblocks. They should probably be removed too.